### DISCUSSION TITLE
Audio Zone issue on iOS

### DISCUSSION THREAD

**Discussion by Interality** [mozilla/hubs#5785] [General Questions & Discussion] [answered]
Hey folks! I'm facing a weird issue on iOS i.e. in the scene where I have used the audio zone. It works intermittently. Tested on Safari & Chrome both. However, works on android well.
Note- Safari: 15.3.1; iOS-15
Is it iOS thing?

**Comment by matthewbcool** [comment, marked as answer]
Thanks for reporting, could be a bug. Could you please share the details in a bug report? https://github.com/mozilla/hubs/issues/new/choose
